src/styles/mixin/card.scss

```diff
@@ -0,0 +1,117 @@
+$page-font-family: Roboto, sans-serif;
+$font-weigth-bolt: 700;
+$font-weigth-normal: 400;
+$primary-color: #060b35;
+$secondary-color: #616070;
+$main-width: 166px;
+$text-color: #fff;
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    font-family: $page-font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-color;
+    align-content: space-evenly;
```

Comment:
The property `align-content` is not applicable to the `.card__description` element as it is not a flex container. Consider using `text-align` or another appropriate property if alignment is needed.